docs/csharp/roslyn-sdk/tutorials/snippets/how-to-write-csharp-analyzer-code-fix/NuGet.config

```diff
@@ -3,7 +3,6 @@
   <packageSources>
     <clear />
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
-    <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
```

Comment:
This is certainly a false positive. The removal of the package will break the sample.

Reply:
@Youssef1313 Can we configure it differently? It's blocking our publishing pipeline.

Reply:
There is currently no way. The appropriate fix should be from roslyn-sdk side when the package is published on nuget.org.